### ISSUE TITLE
b123 - attach to process on machine with same application but modified code

### ISSUE THREAD

**Issue by zadmarbella** [blackberry/VSPlugin#254]
Prerequisite: have 2 machines with the same project solution.
1. make changes to machine 1 project solution
2. launch for debugging
3. attempt attach to process on machine 2

Behaviour:
debugging doesn't work as expected and seg faults observed.

Note: it is not clear if this is supported or not.

**Comment by guarnold** [comment]
I did some tests, building a Win32 and a BB app. Them I modified the source code. 
I started the app and used VS to attach to the running app.

The behavior was the same for both apps, except that, for the Win32 app, a dialog box is presented, informing the user that the source code is different.

I created a new task to implement this feature: #271

**Comment by zadmarbella** [comment]
this issue is still reproducible in v1.01.0022 the feature has yet to be implemented as and is in the backlog. A test has been created to cover for this. Placing in Triage.

**Comment by dbrgss** [comment]
I would recommend closure of this issue.. as it has an unimplemented task associated with it.   A new testing task can be created when the task is finished.
